index.html

```diff
@@ -5,7 +5,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>HTML Sanitizer API Playground</title>
   <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;800&family=Roboto+Slab:wght@100;500&display=swap" rel="stylesheet" crossorigin="anonymous">
-  <link href="/style.css" rel="stylesheet">
+  <link href="style.css" rel="stylesheet">
```

Comment:
(This makes it possible to easily test/debug locally without affecting the online version)